src/app/styles/_hilite.scss

```diff
@@ -28,7 +28,7 @@
   cursor: pointer;
   color: var(--link-color, #27a4fd);
 }
-.clickable .d {
+d {
```

Comment:
`d` isn't a real HTML element, so maybe a span with a class instead? It looks very nice though:

<img width="279" alt="Screen Shot 2020-08-16 at 2 47 01 PM" src="https://user-images.githubusercontent.com/69156/90344527-6b524700-dfcf-11ea-8f33-ce7e0947afa7.png">

Is the idea that these ultimately clickable commands, where you can click it to do the command (in this case "go out"?)

Reply:
Yeah, `<d>` commands mean that the `innerText` is the command whereas some game feed elements declare a `d=` attribute for the command and the `innerText` is different.